modules/contentbox/modules/contentbox-admin/views/twofactor/forceEnrollment.cfm

```diff
@@ -79,7 +79,7 @@
                         Action Bar
                         Saving only if you have permissions, else it is view only.
                     --->
-                    <cfif prc.oCurrentAuthor.checkPermission( "AUTHOR_ADMIN" ) OR prc.author.getAuthorID() EQ prc.oCurrentAuthor.getAuthorID()>
+                    <cfif ( !prc.oCurrentAuthor.isLoaded() AND prc.author.isLoaded() ) OR ( prc.author.getAuthorID() EQ prc.oCurrentAuthor.getAuthorID() OR prc.oCurrentAuthor.checkPermission( "AUTHOR_ADMIN" ) )>
```

Comment:
Why do an is loaded check? You check if not loaded or loaded? Doesn't make sense

Reply:
I added the isLoaded check because at that point the user is not logged in but it is authenticated, so If the user is valid (loaded) I should force to enroll in MFA.